### PR TITLE
Adds draw_rect() variations

### PR DESCRIPTION
**Pull request by MaxLeiter**

### CODE REVIEW

File: src/knightos/display.c [hunk 1]
```diff
@@ -203,6 +203,20 @@ void draw_sprite_xor(SCREEN *screen, unsigned char x, unsigned char y, unsigned
 	screen; x; y; height; sprite;
 }
 
+void draw_rectangle(SCREEN *screen, unsigned char width, unsigned char height, unsigned char x, unsigned char y, GFX_BLEND_MODE blend_mode) {
+	switch (blend_mode) {
+	case GFX_AND:
+		draw_rect_and(screen, width, height, x, y);
+    	break;
```
Comment (SirCmpwn) [inline src/knightos/display.c]: style - indent these

**Comment by SirCmpwn** [comment]
LGTM save for the minor style issue

**Comment by MaxLeiter** [comment]
Bump, went ahead and updated draw_sprite()'s indents as well

File: src/knightos/display.c [hunk 2]
```diff
@@ -119,16 +119,16 @@ void draw_v_line(SCREEN *screen, unsigned char x, unsigned char y, unsigned char
...
-		draw_sprite_and(screen, x, y, height, sprite);
-    	break;
-	case GFX_OR:
-		draw_sprite_or(screen, x, y, height, sprite);
-    	break;
- 	case GFX_XOR:
-		draw_sprite_xor(screen, x, y, height, sprite);
-    	break;
-    }
+		case GFX_AND:
```
Comment (SirCmpwn) [inline src/knightos/display.c]: Sorry - keep the case aligned with the switch. Just bring the break up an indent.

Comment (SirCmpwn) [inline src/knightos/display.c]: That is, line up the break with the body of the case block.

**Comment by SirCmpwn** [comment]
Thanks!

**Pull request merged**